documents/tutorials/contiv-policy.html

```diff
@@ -605,6 +605,192 @@
+b6e0601d8c13        contiv/auth_proxy:1.0.0-beta.6   "./auth_proxy --tls-k"   4 minutes ago       Up 4 minutes                            contiv-node3/auth-proxy
+0c3cb365e573        quay.io/coreos/etcd:v2.3.8       "/etcd"                  7 minutes ago       Up 7 minutes                            contiv-node4/etcd
+a9536ad281be        quay.io/coreos/etcd:v2.3.8       "/etcd"                  8 minutes ago       Up 8 minutes                            contiv-node3/etcd
+
+</code></pre>
+<p>Now run iperf server and client to find out current bandwidth which we are getting on the network
+where you are running this tutorial. It may vary depending upon base OS , network speed etc.</p>
+<pre class="highlight plaintext"><code>On serverA:
+
+[vagrant@contiv-node3 ~]$ docker exec -it serverA sh
+/ # ifconfig
+eth0      Link encap:Ethernet  HWaddr 02:02:32:01:01:01
+          inet addr:50.1.1.1  Bcast:0.0.0.0  Mask:255.255.255.0
+          inet6 addr: fe80::2:32ff:fe01:101%32741/64 Scope:Link
+          UP BROADCAST RUNNING MULTICAST  MTU:1450  Metric:1
+          RX packets:16 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:8 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:1296 (1.2 KiB)  TX bytes:648 (648.0 B)
+
+lo        Link encap:Local Loopback
+          inet addr:127.0.0.1  Mask:255.0.0.0
+          inet6 addr: ::1%32741/128 Scope:Host
+          UP LOOPBACK RUNNING  MTU:65536  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:1
+          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+
+/ # iperf -s -u
+------------------------------------------------------------
+Server listening on UDP port 5001
+Receiving 1470 byte datagrams
+UDP buffer size:  208 KByte (default)
+------------------------------------------------------------
+[  3] local 50.1.1.1 port 5001 connected with 50.1.1.2 port 34700
+[ ID] Interval       Transfer     Bandwidth        Jitter   Lost/Total Datagrams
+[  3]  0.0-10.0 sec  1.25 MBytes  1.05 Mbits/sec   0.028 ms    0/  893 (0%)
+
+
+On clientA:
+
+[vagrant@contiv-node4 ~]$ docker exec -it clientA sh
+/ # iperf -c 50.1.1.1 -u
+------------------------------------------------------------
+Client connecting to 50.1.1.1, UDP port 5001
+Sending 1470 byte datagrams
+UDP buffer size:  208 KByte (default)
+------------------------------------------------------------
+[  3] local 50.1.1.2 port 34700 connected with 50.1.1.1 port 5001
+[ ID] Interval       Transfer     Bandwidth
+[  3]  0.0-10.0 sec  1.25 MBytes  1.05 Mbits/sec
+[  3] Sent 893 datagrams
+[  3] Server Report:
+[  3]  0.0-10.0 sec  1.25 MBytes  1.05 Mbits/sec   0.027 ms    0/  893 (0%)
+/ #
+
+</code></pre>
+<p>Now we see that, current bandwidth we are getting is 1.05 Mbits/sec.
+So let us create new group B and create netprofile with bandwidth less than the one 
```

Comment:
There was an earlier reference to "networkprofile". It is best to use either "netprofile" or "networkprofile" consistently.